packages/next/package.json

```diff
@@ -3,14 +3,19 @@
   "version": "1.1.1",
   "source": "./src/index.ts",
   "main": "./lib/index.js",
-  "exports": {
-    "require": "./lib/index.js",
-    "import": "./lib/module.mjs"
-  },
   "module": "./lib/module.mjs",
   "types": "./lib/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./lib/index.js",
+      "import": "./lib/module.mjs",
+      "types": "./lib/index.d.ts"
+    },
+    "./edge": "./edge/lib/index.js"
```

Comment:
also "import" and "types"?

Reply:
If I add "import" and "types", parcel cannot generate them ("d.js" files), and will cause error for "import".

I can't figure out how to fix it, help needed!